modules/auth/auth.controller.js

```diff
@@ -0,0 +1,81 @@
+const jwt = require("jsonwebtoken");
+const passportJwt = require("passport-jwt");
+const { User } = require("../users/user.schema");
+const { createUser } = require("../users/users.controller");
+
+const login = async (req, res) => {
+  const { email, password } = req.body;
```

Comment:
Zanim pójdziesz dalej warto tutaj zrobić walidacje tych pól. No bo po co wykonywać cokolwiek jeśli np syntax nie zgadza jeśli email nie jest mailem? :)

Reply:
To też jest w zadaniu: "Utwórz walidację wszystkich pól obowiązkowych (email i password). W przypadku błędu walidacji zwróć Błąd walidacji."

Reply:
Dodałam walidację